scripts/bd_to_prod.sh

```diff
@@ -36,29 +36,31 @@ echo ${KEY} | gpg --batch -d --passphrase-fd 0 ${enc_location} > ~/.aws/credenti
 
 echo "Creating lumigo-python-tracer layer"
 ./scripts/prepare_layer_files.sh
-../utils/common_bash/create_layer.sh lumigo-python-tracer ALL python "python3.6 python3.7"
 
 echo "Getting latest changes from git"
 changes=$(git log $(git describe --tags --abbrev=0)..HEAD --oneline)
 
-echo "Creating layer latest version arn table md file (LAYERS.md)"
-cd ../larn && npm i -g
-larn -r python3.6 -n layers/LAYERS36 --filter lumigo-python-tracer -p ~/python_tracer
-larn -r python3.7 -n layers/LAYERS37 --filter lumigo-python-tracer -p ~/python_tracer
-cd ../python_tracer
-git add layers/LAYERS36.md
-git add layers/LAYERS37.md
-git commit -m "layers-table: layers md"
+
 
 sudo pip install --upgrade bumpversion
```

Comment:
same as in node - you're uploading the package to pypi with an old version of the layers/LAYER36.md (which is ok but should be noticed)